.github/workflows/integration_test.yaml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   integration-test:
     name: Integration Test
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
```

Comment:
IS this intended?

Reply:
No, I am just using this pull request to test the new operator-workflow pipeline, I will revert this after I saw the result, thanks!

Reply:
It was a success, removed.